src/main/kotlin/mate/academy/service/FigureSupplier.kt

```diff
@@ -0,0 +1,43 @@
+package mate.academy.service
+
+import mate.academy.model.Color
+import mate.academy.Circle
+import mate.academy.Figure
+import mate.academy.IsoscelesTrapezoid
+import mate.academy.Rectangle
+import mate.academy.RightTriangle
+import mate.academy.Square
+import kotlin.random.Random
+
+// Constants for random dimension range
+const val MIN_RANDOM_DIMENSION = 1
+const val MAX_RANDOM_DIMENSION = 10
+const val NUM_FIGURE_TYPES = 5
+const val VALUE_CIRCLE_RADIUS = 10.0
+const val VALUE_IN_RANDOM_FOUR = 4
+const val VALUE_IN_RANDOM_THREE = 3
+val DIMENSION_RANGE = MIN_RANDOM_DIMENSION..MAX_RANDOM_DIMENSION  // Use 'val' instead of 'const'
+
+class FigureSupplier {
+
+    fun getRandomFigure(): Figure {
+        val colorSupplier = ColorSupplier()
```

Comment:
The `ColorSupplier` class is used here but is not imported. Make sure to import `ColorSupplier` to avoid compilation errors.